core/src/main/java/tachyon/master/TachyonMaster.java

```
@@ -148,6 +150,15 @@ public TachyonMaster(TachyonConf tachyonConf) {
     }
   }
 
+  private TServerSocket createTServerSocket(InetSocketAddress address) throws TTransportException {
+    if (mTachyonConf.getBoolean(Constants.TACHYON_SECURITY_USE_SSL, false)) {
+      // TODO: ssl
+      return null;
```

should throw an unsupported exception.